src/acceptance-test/java/org/zalando/nakadi/webservice/EventStreamReadingAT.java

```diff
@@ -416,29 +429,30 @@ private int countStatusCode(final List<CompletableFuture<HttpURLConnection>> fut
         return future;
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 60000)
```

Comment:
Probably it was an investigation of problem, and I forgot to change it back. Will change it back.